src/Command/DaemonCommand.php

```diff
@@ -123,9 +123,9 @@ private function configureDaemonDefinition(): void
      * Define command code callback.
      *
      * @param callable $callback
-     * @return DaemonCommand
+     * @return $this
      */
-    public function setCode(callable $callback): DaemonCommand
+    public function setCodeDaemon(callable $callback)
```

Comment:
why do we remove the return type?

Reply:
could you also explain a bit more in this PR (maybe send a documentation link) about why we have to change the name of this function, please 🙏

Reply:
This PR is still in progress, I will explain all what I am going to fix when it's all ok, for the moment i'm still testing the best way to make this bundle compatible with projet in Symfony 6 ;-)

Reply:
For this particular point, the setCode method which is a method of symfony/command that daemonBundle extends has been typed :static in Symfony 6 which is incompatible with :DaemonCommand type

Reply:
No more function renaming, we gonna make a special release for symfony 6 and PHP 8